contrib/node/src/python/pants/contrib/node/tasks/node_resolve.py

```diff
@@ -143,6 +143,11 @@ def execute(self):
             if not vt.valid:
               resolver_for_target_type = self._resolver_for_target(target).global_instance()
               resolver_for_target_type.resolve_target(self, target, vt.results_dir, node_paths)
+            if target.is_synthetic:
+              # In the case that the target is synthetic, there could be potentially two “targets”,
```

Comment:
It's not clear this is necessary. The `concrete_derived_from` target here will be a thrift target, so it should not be relevant in a resolve.

Reply:
This is technically not needed, and is duplicate support case for `node-install` task which shouldn't exist since all installs will happen in `pants.d`.